src/UrlParams.ts

```diff
@@ -264,7 +264,7 @@ export const getUrlParams = (
       "skipLobby",
       isWidget && intent === UserIntent.StartNewCall,
     ),
-    returnToLobby: isWidget ? parser.getFlagParam("returnToLobby") : true,
+    returnToLobby: isWidget ? parser.getFlagParam("returnToLobby") : false,
```

Comment:
I don't understand why this is inverted. For SPA we always want to be showing the lobby which is achieved with `returnToLobby` set to `true`.

Reply:
My understanding is: `skipLobby` controls whether we show the lobby when joining the call, which we do want in SPA, and `returnToLobby` controls whether we show the lobby when leaving the call, which we don't want in SPA. The reality is we just don't check the flag at all in SPA mode (the behavior is more or less hardcoded) but, I thought it would be wise to still match its value to the expected behavior.

Reply:
Thank you for the comment to clarify. 👍